src/components/Home.js

```diff
@@ -56,6 +57,7 @@ const Home = () => {
           ))}
         </div>
       </div>
+      <Filter />
```

Comment:
- [ ] Great job implementing a filter component! Can you please put this filter option at the beginning of the page? Also when I choose a country on the filter options, I expect it to be the only visible country listed on the home page, but rather the home page reloads and the data is being duplicated. See the screenshot below as the data starts back from the first country after it had already listed all other countries. Please can you look into this?
![image](https://user-images.githubusercontent.com/60772732/132946501-df616a30-c2fb-43ab-ad3d-c351cd40bc14.png)